source/server/connection_handler_impl.cc

```diff
@@ -148,6 +148,14 @@ void ConnectionHandlerImpl::enableListeners() {
   }
 }
 
+void ConnectionHandlerImpl::setListenerRejectFraction(float reject_fraction) {
+  disable_listeners_ = false;
```

Comment:
seems strange to change this boolean without resuming listening.
Also if new listeners are added won't they be added listening but without a rejection fraction?

Reply:
Thanks, this was a bad copy-paste on my part. Fixed in the latest commit, and added tests.